Anoma/Builtin/ByteArray.juvix

```diff
@@ -12,20 +12,21 @@ axiom mkByteArray : List Byte -> ByteArray;
 builtin bytearray-length
 axiom length : ByteArray -> Nat;
 
--- TODO: Everything below needs to be removed.
-syntax alias AnomaContents := Nat;
+-- TODO Should be moved into base.
+-- TODO Should we use `AnomaEncoded : Type := Nat;` here?
+syntax alias AnomaEncoded := Nat;
```

Comment:
WDYT @paulcadman ?

Reply:
discussed in slack, we came up with `AnomaByteArray` as an alternative.